server/konnectors/edf.coffee

```diff
@@ -1143,20 +1204,42 @@ displayData = (requiredFields, entries, data, next) ->
 
     next()
 
+fetchEdeliaData = (requiredFields, entries, data, next) ->
```

Comment:
No, it will be ok: the [fetcher](https://github.com/cozy-labs/konnectors/blob/master/server/lib/fetcher.coffee) is just a little library build on top of [ware](https://www.npmjs.com/package/ware) to executes a series of tasks with the same signature.
Notification are generated by the konnector model.